src/style-sheets/main.scss

```diff
@@ -13,12 +13,14 @@
 @import './generic//button';
 @import './generic/card';
 @import './generic/card-container';
+@import './generic/card-content';
 @import './generic/feature-link';
 @import './generic/input';
 @import './generic/textarea';
 
 // Components
 @import './components/carousel';
+@import './components/Tag';
```

Comment:
we use kebab casing for sass file names so this file should be lowercase for consistency.